monasca_installer/monasca_config.yml.example

```diff
@@ -166,7 +165,7 @@ smtp_host: localhost
 wsrep_cluster_name: monasca
 
 influxdb_replication_factor: 3
-influxdb_version: 0.8.1
+influxdb_version: 0.9.0-rc
```

Comment:
The only issue I see here is that you need an rc number in this version, for example 0.9.0-rc25. The rc numbers for influxdb have been changing really fast, and are up to pretty high numbers.